src/test/java/org/broadinstitute/hellbender/tools/picard/analysis/QualityScoreDistributionIntegrationTest.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.tools.picard.analysis;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class QualityScoreDistributionIntegrationTest extends CommandLineProgramTest {
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "picard/analysis/QualityScoreDistribution");
+
+    @Test
+    public void test() throws IOException {
+        final File input = new File(TEST_DATA_DIR, "first5000a.bam");
+        final File expectedFile = new File(TEST_DATA_DIR, "qualscoredist.txt");
+        final File outfile = BaseTest.createTempFile("testQualityScoreDistribution", ".metrics");
+        final File pdf = BaseTest.createTempFile("testQualityScoreDistribution", ".pdf");
+        outfile.deleteOnExit();
+        pdf.deleteOnExit();
```

Comment:
No need to set `deleteOnExit()` -- `BaseTest.createTempFile()` does this for you.